src/main/java/edu/kis/powp/jobs2d/command/CircleFactory.java

```diff
@@ -0,0 +1,23 @@
+package edu.kis.powp.jobs2d.command;
+
+import edu.kis.powp.jobs2d.drivers.DriverManager;
+
+public class CircleFactory {
+
+    public static ComplexCommand createRectangle(DriverManager driverManager) {
```

Comment:
DriverCommand return type would be sufficient
Passing a manager instead of a driver increases dependencies.